edit/edit.go

```diff
@@ -59,7 +58,7 @@ func ParseLabel(target string) (string, string, string) {
 	if len(parts) == 1 {
 		if strings.HasPrefix(target, "//") || tables.StripLabelLeadingSlashes {
 			// "//absolute/pkg" -> "absolute/pkg", "pkg"
-			return repo, parts[0], path.Base(parts[0])
+			return repo, parts[0], filepath.Base(parts[0])
```

Comment:
Why `filepath`? Labels always have forward slashes regardless of current operating system.

Reply:
Ah, you're right. It seems I went a bit overboard when I fixed file path handling for windows.

Reverting 👌